src/interface.js

```diff
@@ -0,0 +1,66 @@
+var scoreCard = new ScoreCard();
+$( document ).ready(function() {
+
+  var showRollScores = function() {
+    $("#rolls").html(scoreCard.rollArray);
+    $("#frameScores").html(scoreCard.frameScores);
+    $("#totalScore").html(scoreCard.scoreTotal);
+
+  };
+
+  $("#0").click(function() {
```

Comment:
Very repetitive!

If you give your buttons a class, listen for clicks on that class, then use this.id to know which button was clicked on, you can reduce these down to a single listener.
=)